ivy/functional/ivy/creation.py

```diff
@@ -300,13 +300,66 @@ def full_like(
     but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
     instances in place of any of the arguments.
 
-    Examples
-    --------
-    >>> x = ivy.array([1, 2, 3, 4, 5, 6])
-    >>> fill_value = 1
-    >>> y = ivy.full_like(x, fill_value)
-    >>> print(y)
-    ivy.array([1, 1, 1, 1, 1, 1])
+    Functional Examples
+    -------------------
+    With int datatype:
+
+        >>> import ivy
+        >>> x = ivy.array([1, 2, 3, 4, 5, 6])
+        >>> fill_value = 1
+        >>> y = ivy.full_like(x, fill_value)
+        >>> print(y)
+        ivy.array([1, 1, 1, 1, 1, 1])
+        >>> fill_value = 0.000123
+        >>> x = ivy.ones(5)
+        >>> y = ivy.full_like(x , fill_value)
+        >>> print(y)
+        ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+    with float datatype:
+
+        >>> import ivy
+        >>> x = ivy.array([1.0, 2.0, 3.0, 4.0, 5.0, 6.0]) # for filling values with floating datatype the array is required to be of same dtype (float in this case)
+        >>> fill_value = 0.000123
+        >>> y = ivy.full_like(x, fill_value)
+        >>> print(y)
+        ivy.array([0.000123, 0.000123, 0.000123, 0.000123, 0.000123, 0.000123])
+
+    with torch.Tensor input:
```

Comment:
this is not needed as torch tensors are part of native array examples, it does not need to have examples on its own